Drivers/Src/Minions.c

```diff
@@ -84,6 +169,65 @@ uint16_t Lights_Bitmap_Read(light_t light) {
     return lightStatesBitmap;
 }
 
+/**
+ * @brief   Reads from static variable bitmap holding values of switches
+ * @param   sw
+ * @return  ON/OFF State
+ */ 
+State Switches_Read(switches_t sw){
+    return (State) ((SwitchStates_Bitmap >> sw) & 0x0001);
```

Comment:
This same enum typecast should be happening on line 160 as well

Reply:
Fixed.